src/main/java/org/example/Pair.java

```diff
@@ -0,0 +1,71 @@
+package org.example;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Pair<K, V> {
+    public static void main(String[] args) {
```

Comment:
nie jest dobrą praktyką trzymanie metody main w klasach które implementują jakąś funkcjonalność. Najlepiej zamiast takiej metody main zrobić testy jednostkowe, a jeśli już chcesz tą metode main to trzymaj ją odseparowaną w innej klasie